scanspec/specs.py

```diff
@@ -75,13 +75,13 @@ class Product(Spec):
         spec = Line("y", 1, 2, 3) * Line("x", 3, 4, 12)
```

Comment:
Should turn `positions()` into `midpoints()`

Reply:
And `SpecPositions` to `Midpoints`

Reply:
Updated as suggested